mlir/python/mlir/dialects/_affine_ops_ext.py

```diff
@@ -0,0 +1,54 @@
+#  Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+#  See https://llvm.org/LICENSE.txt for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+try:
+    from ..ir import *
+    from ._ods_common import get_op_result_or_value as _get_op_result_or_value
+    from ._ods_common import get_op_results_or_values as _get_op_results_or_values
+except ImportError as e:
+    raise RuntimeError("Error loading imports from extension module") from e
+
+from typing import Optional, Sequence, Union
+
+
+class AffineStoreOp:
+    """Specialization for the Affine store operation."""
+
+    def __init__(
+        self,
+        value: Value,
```

Comment:
This should also be a union so we can take an operation and "unpack" it to get the result (if single). Similarly to the other argument. Note that the implementation handles this correctly.

Reply:
Fixed to be the same as the other argument (i.e. Union[Operation, OpView, Value]).  Thank you!